apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/RpcRetryConsumerTest.cs

```diff
@@ -210,11 +208,11 @@ private void SetupTestReceiveThrows(Action<FakeSequentialThreadingTimer, Mock<IC
         /// </summary>
         private class FakeSequentialThreadingTimer : ISequentialThreadingTimer
         {
-            private Action _callback;
+            internal Action Callback = null;
```

Comment:
you can still make it a property, right?  having a public field like that is usually not done.  I know its just a fake, but this should work just as well:

internal Action Callback { get; set; }

also, initializing to null is probably redundant.